nighthawk/source/client/output_formatter_impl.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "nighthawk/client/output_formatter.h"
+
+#include "envoy/common/time.h"
+
+#include "common/protobuf/protobuf.h"
+
+#include <cstdint>
+
+namespace Nighthawk {
+namespace Client {
+
+class OutputFormatterImpl : public OutputFormatter {
+public:
+  OutputFormatterImpl(Envoy::TimeSource& time_source, const Options& options,
+                      const std::vector<StatisticPtr>& merged_statistics,
```

Comment:
Can you add some comments on what "merged" means here for stats/counters?

Reply:
done